deployer/scripts/upgrade.sh

```diff
@@ -58,6 +58,12 @@ function getDeploymentVersion() {
     return
   fi
 
+  # check for ES MIN_MASTER env var
+  if [ -z "$(oc get configmap logging-elasticsearch -o yaml | grep 'MIN_MASTERS')" ]; then
```

Comment:
This means "if MIN_MASTERS is not in the configmap, return 8" - is this as intended?